### PR TITLE
Feature: Deprioritize booster RCs after 90 days (EXPOSUREAPP-12366)

### PR DESCRIPTION
**Pull request by mlenkeit**
Related to:
- https://github.com/corona-warn-app/cwa-app-ccl/issues/49

### CODE REVIEW

File: test/fixtures/ccl/dcc-series-recovery.yaml [hunk 1]
```diff
@@ -525,13 +525,13 @@
...
         admissionState: 2G
-        mostRelevantCertificate: rc1
+        mostRelevantCertificate: biontech1/2
         vaccinationState: COMPLETE_IMMUNIZATION
         vaccinationValidFrom: 
         mostRecentVaccination: biontech1/2
         hasBooster: false
         verificationCertificates:
-          - certificate: rc1
+          - certificate: biontech1/2
```
Comment (Ein-Tim) [inline test/fixtures/ccl/dcc-series-recovery.yaml:534]: In this case it's difficult because non of the certificates are valid with CovPassCheck, so it actually does not matter whether the recovery or the unfinished vaccination certificate is prioritized.

I'm wondering if it would make sense to keep the recovery certificate as the most relevant one here as it technically can be valid in other countries (e.g. Italy) while the vaccination certificate is not valid anywhere.

Comment (mlenkeit) [reply test/fixtures/ccl/dcc-series-recovery.yaml:534]: @Ein-Tim we came across the same thing internally and concluded that this is rather an edge case and handling this the way you propose is relatively complicated. That's why we decided to prioritize 1/2 then.

Comment (Ein-Tim) [reply test/fixtures/ccl/dcc-series-recovery.yaml:534]: Yeah, I know. It's definitely fine for me how it is now!
